weave

```diff
@@ -205,6 +205,40 @@ dns_arg_count() {
     fi
 }
 
+collect_rewrite_hosts_args() {
+    REWRITE_HOSTS_ARG_COUNT=0
+    case "$1" in
+    --rewrite-hosts)
+        REWRITE_HOSTS=1
+        REWRITE_HOSTS_ARG_COUNT=1
+        ;;
```

Comment:
It's being used [here](https://github.com/weaveworks/weave/commit/da9597d9d830ec92b0586e6ee2dc917d885bc241#diff-ed6579eaedd0e67b3ed51c7ee7071044R279) too.